test/unit/test_rtp.py

```diff
@@ -0,0 +1,247 @@
+import pydarn
+import unittest
+import matplotlib.pyplot as plt
+from datetime import datetime
+
+
+class TestRTP(unittest.TestCase):
+    """
+    Testing class RTP, due to the plotting nature all plots will be saved
+    since there is return testing
+    """
+    @classmethod
+    def setUpClass(cls):
+        """
+        Runs once before all tests are ran. Loads and reads in the
+        fitacf file that will be used in all the unit tests.
+        """
+        #fitacf_file = "../testfiles/20190203.0001.00.fhw.fitacf3"
+        fitacf_file = "../testfiles/20180220.C0.rkn.fitacf"
+        darn_read = pydarn.DarnRead(fitacf_file)
+        cls.fitacf_data = darn_read.read_fitacf()
+
```

Comment:
Do you need a tearDown function?

Reply:
tearDown is good for disconnecting connections to databases or closing files if you open them. However; I do not necessarily need one because we don't have any open file sockets in this case. I could have one to delete the data but python's garbage collector should do that. I was thinking if we saved images we could make some tearDown type function such that the images don't clutter up the folder.